packages/py-api/py_api/routes/uploader_routes.py

```diff
@@ -16,6 +16,6 @@ async def upload_object(file: UploadFile, filename: Annotated[str, Form()]) -> D
     return c.upload_object(file, filename)
 
 
-@router.delete("")
-async def delete_object() -> Dict[str, Any]:  # type: ignore
-    pass
+@router.delete("/{image_name:path}")
```

Comment:
Why is this `image_name:path` and not just `image_name`?
Also, I believe `filename` is better since that's what we call the name of the file in the other endpoint.